example/index.js

```diff
@@ -26,11 +26,13 @@ class App extends Component {
     return (
       <main>
         <Flatpickr data-enable-time className='test'
-          onChange={(_, str) => console.info(str)} />
+          options={{onChange: (_, str) => console.info(str)}} />
```

Comment:
Sorry, I mean.

I want to use `Flatpickr` like this

```jsx
<Flatpickr onChange={...} options={...} />
```

For most use cases, maybe we don't need pass options.
We should `alias` `props.onChange` to `props.options.onChange`